queries.sql

```diff
@@ -46,3 +46,74 @@ select full_name, count(name) from owners left join animals on animals.owner_id=
 select animals.name from owners join animals on animals.owner_id= owners.id where owners.full_name='Jennifer Orwell' and animals.species_id=2;
 select animals.name from owners join animals on animals.owner_id= owners.id where owners.full_name='Dean Winchester' and animals.escape_attempts=0;
 select species.name, count(animals.name) from species join animals on animals.species_id= species.id group by species.name;
+
+
+-- Who was the last animal seen by William Tatcher?
+SELECT animals.name 
+  FROM animals
+  JOIN  visits ON  animals.id = animals_id 
+  JOIN vets ON vets_id = vets.id WHERE vets.name = 'William Tatcher'
+  ORDER BY date_of_visit DESC LIMIT 1;
+
+-- How many different animals did Stephanie Mendez see?
+SELECT COUNT(DISTINCT animals.name)
+  FROM animals
+  JOIN  visits ON  animals.id = animals_id 
+  JOIN vets ON vets_id = vets.id WHERE vets.name = 'Stephanie Mendez';
+
```

Comment:
- [ ] It would be better if we improve our README file to be descriptive, we can fix this by changing the current header and description to the project